test/synthetics/latest_release_cdn.js

```diff
@@ -0,0 +1,144 @@
+    }];
+    if (arm64Unsupported.indexOf(repo) < 0){
+      archs.push({ 
+        archInUrl: 'x86_64', 
+        archInPackage: repo === 'nrjmx' ? 'noarch' : 'x86_64'
+      });
+    } 
+
+    // except from infra-agent, all OHIs have consistent repos and package names.
+    var packagePrefix = repo === 'infrastructure-agent' ? 'newrelic-infra' : repo;
+
+    // TODO: add checks for all OS versions (blocked until all OHIs are released) 
+    // TODO: add DockerHub images checks
+    checkSLES(packagePrefix, latestRelease, archs);
+    checkAL2(packagePrefix, latestRelease, archs);
+    checkRHEL(packagePrefix, latestRelease, archs);
+    checkDeb(packagePrefix, latestRelease, archs);
+
+    if (windowsUnsupported.indexOf(packagePrefix) < 0){
+      checkWindows(packagePrefix, latestRelease);
+    }
+}
+
+function checkSLES(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}-${latestRelease}-1.sles12.4.${archs[i].archInPackage}.rpm` : 
+         `${packagePrefix}-${latestRelease}-1.${archs[i].archInPackage}.rpm`;
+    var downloadServerURI = `${downloadServerBaseURI}/linux/zypp/sles/12.4/${archs[i].archInUrl}/${packageStructure}`; 
+    checkFileInServer(downloadServerURI);
+  }
+}
+
+function checkAL2(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}-${latestRelease}-1.amazonlinux-2.${archs[i].archInPackage}.rpm` : 
+      `${packagePrefix}-${latestRelease}-1.${archs[i].archInPackage}.rpm`;
+    
+    var downloadServerURI = `${downloadServerBaseURI}/linux/yum/amazonlinux/2/${archs[i].archInUrl}/${packageStructure}`;
+    checkFileInServer(downloadServerURI);
+  }
+}
+
+function checkRHEL(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}-${latestRelease}-1.el8.${archs[i].archInPackage}.rpm` :
+      `${packagePrefix}-${latestRelease}-1.${archs[i].archInPackage}.rpm`;
+    
+    var downloadServerURI = `${downloadServerBaseURI}/linux/yum/el/8/${archs[i].archInUrl}/${packageStructure}`; 
+    checkFileInServer(downloadServerURI);
+  }
+}
+
+function checkDeb(packagePrefix, latestRelease, archs){
+  for (var i=0; i < archs.length; i++){
+    var arch = archs[i].archInPackage === 'x86_64' ? 'amd64' : archs[i].archInPackage;
+    var packageStructure = packagePrefix === 'newrelic-infra' ? `${packagePrefix}_systemd_${latestRelease}_${arch}.deb` : 
+      `${packagePrefix}_${latestRelease}-1_${arch}.deb`;
+    
+    var downloadServerURI = `${downloadServerBaseURI}/linux/apt/pool/main/n/${packagePrefix}/${packageStructure}`; 
+    checkFileInServer(downloadServerURI);
+  }
+}
+
+/**
+* Agent URL example: 
+*  - https://download.newrelic.com/infrastructure_agent/windows/newrelic-infra.1.20.4.msi
```

Comment:
It would be nice to check update time or better checksum/size for "latest" msi

Reply:
good point, since this is a just a binary, how could we check this without downloading the file?

Reply:
I think you have to download it to check sha, so we can try to check upload time in s3 xml. If you need any help let me know